packages/@tinacms/toolkit/src/packages/fields/components/Reference.tsx

```diff
@@ -92,7 +92,7 @@ export const Reference: React.FC<ReferenceProps> = ({
   options,
 }) => {
   const cms = useCMS()
-  const hasTinaAdmin = cms.flags.get('tina-admin')
+  const hasTinaAdmin = cms.flags.get('tina-admin') || true
```

Comment:
If the user has `cms.flags.get('tina-admin')` set to `false` this would evaluate to `true`